[Bug]: Broken UTs on `main` branch

### Describe the bug

Seeing failures in these 2 UTs on main branch 

```
Run pipenv run coverage run -m pytest --cov=./src --cov-report=xml
F....................................................................... [ 25%]
........................................................................ [ 50%]
........................................................................ [ 76%]
....................................................................     [100%]
=================================== FAILURES ===================================
__________________________ TestRunAssemble.test_main ___________________________

self = <tests.test_run_assemble.TestRunAssemble testMethod=test_main>
mock_copy = <MagicMock name='copy2' id='140398777086032'>
mock_temp = <MagicMock name='TemporaryDirectory' id='140398777095120'>
mock_recorder = <MagicMock name='BundleRecorder' id='140398777104272'>
mock_bundles = <MagicMock name='Bundles' id='140398777121680'>
mocks = (<MagicMock name='getcwd' id='140398777147472'>, <MagicMock name='makedirs' id='140398777161040'>, <MagicMock name='chdir' id='140398777178384'>)
mock_bundle = <MagicMock name='Bundles.create()' id='140398776678672'>

    @patch("os.chdir")
    @patch("os.makedirs")
    @patch("os.getcwd", return_value="curdir")
    @patch("argparse._sys.argv", ["run_assemble.py", BUILD_MANIFEST])
    @patch("run_assemble.Bundles", return_value=MagicMock())
    @patch("run_assemble.BundleRecorder", return_value=MagicMock())
    @patch("tempfile.TemporaryDirectory")
    @patch("shutil.copy2")
    def test_main(self, mock_copy, mock_temp, mock_recorder, mock_bundles, *mocks):
        mock_temp.return_value.__enter__.return_value = tempfile.gettempdir()
        mock_bundle = MagicMock(archive_path="path")
        mock_bundles.create.return_value = mock_bundle
    
>       main()

tests/test_run_assemble.py:47: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
src/run_assemble.py:48: in main
    exit(1)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = Use exit() or Ctrl-D (i.e. EOF) to exit, code = 1

    def __call__(self, code=None):
        # Shells like IDLE catch the SystemExit, but listen when their
        # stdin wrapper is closed.
        try:
            sys.stdin.close()
        except:
            pass
>       raise SystemExit(code)
E       SystemExit: 1

/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/_sitebuiltins.py:26: SystemExit
------------------------------ Captured log call -------------------------------
ERROR    root:run_assemble.py:46 No installation script found at path: /home/runner/work/opensearch-build/opensearch-build/release/tar/linux/opensearch-tar-install.sh

---------- coverage: platform linux, python 3.7.12-final-0 -----------
Coverage XML written to file coverage.xml

=========================== short test summary info ============================
FAILED tests/test_run_assemble.py::TestRunAssemble::test_main - SystemExit: 1
1 failed, 283 passed in 31.93s
Error: Process completed with exit code 1.
```

### To reproduce

Run ut coverage analysis on repo `opensearch-build`, branch `main`

```
pipenv run coverage run -m pytest --cov=./src --cov-report=xml
```

### Expected behavior

No UT failures

### Screenshots

If applicable, add screenshots to help explain your problem.

### Host / Environment

_No response_

### Additional context

_No response_

### Relevant log output

_No response_

--- a/.github/workflows/tests.yml
+++ b/.github/workflows/tests.yml
@@ -1,4 +1,4 @@
-name: bundle-workflow
+name: tests
 
 on:
   push:

--- a/DEVELOPER_GUIDE.md
+++ b/DEVELOPER_GUIDE.md
@@ -8,7 +8,7 @@
     - [Yarn](#yarn)
   - [Install Dependencies](#install-dependencies)
   - [Run Tests](#run-tests)
-  - [Run bundle-workflow](#run-bundle-workflow)
+  - [Build OpenSearch](#build-opensearch)
   - [Code Linting](#code-linting)
   - [Type Checking](#type-checking)
   - [Code Coverage](#code-coverage)
@@ -24,11 +24,15 @@ Fork this repository on GitHub, and clone locally with `git clone`.
 
 #### Pyenv
 
-Use pyenv to manage multiple versions of Python. This can be easily installed with [pyenv-installer](https://github.com/pyenv/pyenv-installer).
+Use pyenv to manage multiple versions of Python. This can be installed with [pyenv-installer](https://github.com/pyenv/pyenv-installer).
+
+```
+curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
+```
 
 #### Python 3.7
 
-Python projects in this repository, including the [bundle-workflow](./bundle-workflow) project, use Python 3.7. See the [Python Beginners Guide](https://wiki.python.org/moin/BeginnersGuide) if you have never worked with the language. 
+Python projects in this repository use Python 3.7. See the [Python Beginners Guide](https://wiki.python.org/moin/BeginnersGuide) if you have never worked with the language.
 
 ```
 $ python3 --version
@@ -38,7 +42,8 @@ Python 3.7.11
 If you are using pyenv.
 
 ```
-$ pyenv install 3.7.11
+pyenv install 3.7.12
+pyenv global 3.7.12
 ```
 
 #### Pipenv
@@ -46,6 +51,8 @@ $ pyenv install 3.7.11
 This project uses [pipenv](https://pipenv.pypa.io/en/latest/), which is typically installed with `pip install --user pipenv`. Pipenv automatically creates and manages a virtualenv for your projects, as well as adds/removes packages from your `Pipfile` as you install/uninstall packages. It also generates the ever-important `Pipfile.lock`, which is used to produce deterministic builds.
 
 ```
+$ pip install pipenv
+
 $ pipenv --version
 pipenv, version 19.0
 ```
@@ -86,7 +93,7 @@ $ pipenv run pytest
 2 passed in 02s
 ```
 
-### Run bundle-workflow
+### Build OpenSearch
 
 Try running `./build.sh`. It should complete and show usage.
 
@@ -134,7 +141,7 @@ All done! ✨ 🍰 ✨
 23 files left unchanged.
 ```
 
-If your code isn't properly formatted, don't worry, [a CI workflow](./github/workflows/bundle-workflow.yml) will make sure to remind you. 
+If your code isn't properly formatted, don't worry, [a CI workflow](./github/workflows/tests.yml) will make sure to remind you. 
 
 ### Type Checking
 

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 <img src="https://opensearch.org/assets/brand/SVG/Logo/opensearch_logo_default.svg" height="64px"/>
 
-[![tests](https://github.com/opensearch-project/opensearch-build/actions/workflows/bundle-workflow.yml/badge.svg)](https://github.com/opensearch-project/opensearch-build/actions/workflows/bundle-workflow.yml)
+[![tests](https://github.com/opensearch-project/opensearch-build/actions/workflows/tests.yml/badge.svg)](https://github.com/opensearch-project/opensearch-build/actions/workflows/tests.yml)
 [![manifests](https://github.com/opensearch-project/opensearch-build/actions/workflows/manifests.yml/badge.svg)](https://github.com/opensearch-project/opensearch-build/actions/workflows/manifests.yml)
 [![codecov](https://codecov.io/gh/opensearch-project/opensearch-build/branch/main/graph/badge.svg?token=03S5XZ80UI)](https://codecov.io/gh/opensearch-project/opensearch-build)
 

--- a/scripts/README.md
+++ b/scripts/README.md
@@ -1,19 +1,11 @@
 - [Scripts](#scripts)
-  - [Bundle Workflow](#bundle-workflow)
-  - [Legacy Scripts](#legacy-scripts)
     - [Run Deployment Script](#run-deployment-script)
       - [Requirements](#requirements)
       - [Usage](#usage)
   
 ## Scripts
 
-### Bundle Workflow
-
-This folder contains default and custom scripts located by [src/paths/script_finder.py](ScriptFinder).
-
-### Legacy Scripts
-
-This folder contains the following scripts which are used in either tar/docker or legacy github actions.
+This folder contains default and custom scripts located by [src/paths/script_finder.py](ScriptFinder), and the following scripts which are used in either tar/docker or legacy github actions.
 
 #### Run Deployment Script
 

--- a/scripts/legacy/tar/linux/opensearch-tar-build.sh
+++ b/scripts/legacy/tar/linux/opensearch-tar-build.sh
@@ -153,7 +153,7 @@ mkdir -p $WORKING_DIR/data
 chmod 755 $WORKING_DIR/data/
 
 # Copy the tarball installation script 
-cp $REPO_ROOT/release/tar/linux/opensearch-tar-install.sh $WORKING_DIR/
+cp $REPO_ROOT/scripts/legacy/tar/linux/opensearch-tar-install.sh $WORKING_DIR/
 
 # Setup k-NN-library
 mkdir -p $WORKING_DIR/plugins/opensearch-knn/knnlib

--- a/src/run_assemble.py
+++ b/src/run_assemble.py
@@ -38,7 +38,7 @@ def main():
     tarball_installation_script = os.path.realpath(
         os.path.join(
             os.path.dirname(os.path.abspath(__file__)),
-            "../release/tar/linux/opensearch-tar-install.sh",
+            "../scripts/legacy/tar/linux/opensearch-tar-install.sh",
         )
     )
     if not os.path.isfile(tarball_installation_script):

--- a/tests/test_run_assemble.py
+++ b/tests/test_run_assemble.py
@@ -52,7 +52,7 @@ class TestRunAssemble(unittest.TestCase):
             os.path.realpath(
                 os.path.join(
                     os.path.dirname(os.path.abspath(__file__)),
-                    "../release/tar/linux/opensearch-tar-install.sh",
+                    "../scripts/legacy/tar/linux/opensearch-tar-install.sh",
                 )
             ),
             "path/opensearch-tar-install.sh",

